apps/rush-lib/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Change Log - @microsoft/rush-lib
 
-This log was last generated on Fri, 21 Jul 2017 22:34:44 GMT and should not be manually modified.
+This log was last generated on Wed, 30 Aug 2017 18:46:19 GMT and should not be manually modified.
+
+## 3.0.15
+Wed, 30 Aug 2017 18:24:39 GMT
+
+### Patches
+
+- Add tar as a dependency of Rush which is required work for replacing the temp_modules/*/package.json files with TGZ files
+- Add repositoryUrl to RushConfiguration to track remote repository
+- Use the new Json API from node-core-library
+- Add two new methods to ChangeFile class
+- Introduce an experimental "rush version" action to manage project versions based on version policy
+- Make "rush generate" not throw if there is a problem reading the shrinkwrap file
```

Comment:
We should not be duplicating changelogs between rush-lib and rush.

Either:
1. Rush-lib is just "the API for Rush", so its log should only say "Rush 1.2.3 was released".  All the actual change logs should go in rush, even if they are API changes, because Rush is one thing.  <--- this is my vote

2. Document the API changes in rush-lib, and the tool feature changes in rush.  This is a reasonable approach, but more work than # 1, with arguably dubious benefits.

@qz2017 @nickpape-msft @iclanton

Reply:
We have been duplicating the change log for both rush and rush-lib. 

We will enable a version policy on rush and rush-lib so that changes in those two projects can trigger rush change but do not get version bump. Then we can keep change log for rush and rush-lib separate w/o extra manual work.

---
In reply to: [136168865](https://github.com/Microsoft/web-build-tools/pull/338#discussion_r136168865) [](ancestors = 136168865)